frame/staking/src/pallet/mod.rs

```diff
@@ -141,6 +148,31 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxNominatorRewardedPerValidator: Get<u32>;
 
+		/// The maximum number of unapplied slashes to be stored in `UnappliedSlashes`
+		#[pallet::constant]
+		type MaxUnappliedSlashes: Get<u32>;
+
+		/// The maximum number of invulnerables, we expect no more than four invulnerables and
+		/// restricted to testnets.
+		#[pallet::constant]
+		type MaxNbOfInvulnerables: Get<u32>;
+
+		/// Maximum number of eras for which the stakers behind a validator have claimed rewards.
+		#[pallet::constant]
+		type MaxErasForRewards: Get<u32>;
+
+		/// Maximum number of validators.
+		#[pallet::constant]
+		type MaxNbOfValidators: Get<u32>;
```

Comment:
```suggestion
		type MaxValidators: Get<u32>;
```

Reply:
Renamed to `MaxValidatorsCount` which coincides with the previous name given to it when it was still a storage item.